chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -561,6 +578,7 @@ public void streamSEDDetailsDocumentContents(final HttpServletResponse response)
                     + "authority on the ACB that certified the product.")
     @RequestMapping(value = "/{certifiedProductId}", method = RequestMethod.PUT,
     produces = "application/json; charset=utf-8")
+    @CacheControl(policy = CachePolicy.PUBLIC, maxAge = CacheMaxAge.TWELVE_HOURS)
```

Comment:
Is this right for an update to be cached? Seems a little odd. If it's right, I think that the update listing method adds a custom header which indicates to the UI that the CHPL ID may have changed and causes the little orange popup in the top right of the screen to show up. I was just wanting to ensure you tested that still works since the CacheControl annotation is fussing with the response headers.

Reply:
This was a mistake - copied the code to the wrong method...